ray_data_eval/video_inference/ray_data_pipeline.py

```diff
@@ -1,27 +1,56 @@
+import argparse
+import csv
 import functools
 import io
 import time
 
+# import json
+
 import humanize
 import numpy as np
 import ray
 from ray.data.block import DataBatch
-from ray._private import profiling
+
+# from ray._private import profiling
 import torch
 from transformers import (
     VideoMAEImageProcessor,
     VideoMAEForVideoClassification,
 )
 
+from ray_data_pipeline_helpers import ChromeTracer, append_gpu_timeline, download_train_directories
+
+parser = argparse.ArgumentParser(description="PyTorch ImageNet Training")
+parser.add_argument(
+    "-s",
+    "--source",
+    metavar="SOURCE",
+    default="local",
+    help="local or S3",
+)
+
+args = parser.parse_args()
+data_source = args.source
```

Comment:
remove this and just use `args.source`. In general reduce the number of global vars.